boards/arm/s32z270dc2_r52/s32z270dc2_r52.dtsi

```diff
@@ -4,7 +4,6 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#include <arm/nxp/nxp_s32z27x_r52.dtsi>
```

Comment:
Was this change intentional?

Reply:
It was intentional, but I will open another PR to fix it. This header is being included twice in the chain of includes. No blocking for these changes so removed.